### ISSUE TITLE
Edge table references a table that is not defined as a vertex table

### ISSUE THREAD

**Issue by Dtenwolde** [cwida/duckdb-pgq#35]
What should happen in the following case: 
```sql
CREATE TABLE person(id BIGINT, name VARCHAR);
CREATE TABLE know(src BIGINT, dst BIGINT, createDate BIGINT);
CREATE TABLE employer(id BIGINT);


CREATE PROPERTY GRAPH pg
VERTEX TABLES (
    person PROPERTIES ( id, name ) LABEL Person,
    )
EDGE TABLES (
    employs    SOURCE KEY ( src ) REFERENCES employer ( id )
            DESTINATION KEY ( dst ) REFERENCES Student ( id )
            PROPERTIES ( createDate ) LABEL Employs
    )
```

In this case the table `employer` is not defined as a vertex table. Therefore this should not be valid? 
Reading the docs the definition of an edge table includes: 
`The name of the destination vertex table.` [p.12]

But in this case, the table referenced is not a vertex table as defined on page 11.

**Comment by Dtenwolde** [comment]
This should not be allowed. 
Oracle 23c provides the following error (which makes sense)
```
DESTINATION KEY (u) REFERENCES university (id)
                                           *
ERROR at line 13:
ORA-42405: Referenced vertex table UNIVERSITY does not exist.
```

**Comment by Dtenwolde** [comment]
Fixed as of #92f0bfd
